include/rocksdb/db.h

```diff
@@ -412,6 +415,32 @@ class DB {
     return Get(options, DefaultColumnFamily(), key, value);
   }
 
+  // Get() methods that return timestamp. Derived DB classes don't need to worry
+  // about this group of methods if they don't care about timestamp feature.
+  virtual inline Status Get(const ReadOptions& options,
```

Comment:
Any particular reason for `inline`?

Reply:
It followed the same pattern with the existing Get() methods.